server/chat-plugins/calculator.ts

```diff
@@ -141,11 +151,11 @@ export const commands: ChatCommands = {
 	calculate(target, room, user) {
 		if (!target) return this.parse('/help calculate');
 		let base = 10;
-		if (target.includes('0x')) {
+		if (target.includes('0x') || target.includes('h')) {
 			base = 16;
-		} else if (target.includes('0o')) {
+		} else if (target.includes('o')) {
```

Comment:
This would be a problem if the calculator were ever expanded to support more functions (`cos` comes to mind; this would make all expressions involving cosines use base 8). I don't think we're planning to do that, but it's worth noting.

Reply:
```suggestion
		} else if (/\bo\b/.test(target)) {
```

Reply:
Probably the easiest way to prevent that false positive.

Reply:
Actually, in hindsight, that's not a good approach, either.